ops/services/app_service/metabase/main.tf

```diff
@@ -1,3 +1,75 @@
+  }
+
+  schema_name = "public"
+
+  grant_command = <<EOF
+  psql \
+  --host ${var.postgres_server_name} \
+  --username ${var.administrator_login} \
+  --dbname "${var.postgres_db_name}" \
+  --command "
+    REVOKE ALL PRIVILEGES ON SCHEMA public FROM PUBLIC;
+    CREATE ROLE ${data.azurerm_key_vault_secret.postgres_nophi_user.value} with LOGIN ENCRYPTED PASSWORD
+    '${data.azurerm_key_vault_secret.postgres_nophi_pass.value}';
+    GRANT USAGE ON SCHEMA ${local.schema_name} TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE facility_device_type TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE device_type TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE organization TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE patient_answers TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE facility TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE patient_link TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE data_hub_upload TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE time_of_consent TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE specimen_type TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE device_specimen_type TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE facility_device_specimen_type TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    GRANT SELECT ON TABLE test_order TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+
+    GRANT SELECT (login_email, internal_id, created_at, updated_at, last_seen, is_deleted) ON TABLE api_user
+    TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+    
+    GRANT SELECT (internal_id, created_at, created_by, updated_at, updated_by, is_deleted, provider_id)
+    ON TABLE provider TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+                
+    GRANT SELECT (internal_id, created_at, created_by, updated_at, updated_by, is_deleted, organization_id, employed_in_healthcare,
+    resident_congregate_setting, facility_id, race, gender, ethnicity, lookup_id, role)
+    ON TABLE person TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+                
+    GRANT SELECT (device_specimen_type_id, created_at, created_by, updated_at, updated_by, patient_id, organization_id, 
+    device_type_id, result, facility_id, survey_data, date_tested_backdate, test_order_id, correction_status, 
+    prior_corrected_test_event_id, internal_id, reason_for_correction)
+    ON TABLE test_event TO ${data.azurerm_key_vault_secret.postgres_nophi_user.value};
+  "
+  EOF
+}
+
+variable "app_settings_overrides" {
```

Comment:
I should have said this on the previous review, but let's move this to `./_vars.tf`

Reply:
Sounds good, done.